src/lib/pmd/PmdEngine.ts

```diff
@@ -447,7 +447,7 @@ export class PmdEngine extends BasePmdEngine {
 	 * a list of rules.  Ideally we could pass in rules, like with other engines, filtered ahead of time by
 	 * the catalog.  If that ever happens, we can remove the ruleGroups argument and use the rules directly.
 	 */
-	/* eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars */
+	/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
Typescript rule overrides the vanilla rule, so we only need to disable the former.